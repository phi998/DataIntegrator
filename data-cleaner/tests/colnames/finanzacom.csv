Article Content,Article Content,Author,Publication Date,Article Content,Article Content,Publication Date,Article Content,Publication Date,Article Content,Publication Date,Article Content
