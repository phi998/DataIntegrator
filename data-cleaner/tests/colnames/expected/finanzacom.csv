Title,Title,Author,Publication Date,Article Content,.,.,.,.,.,.,.
